protocol/x/bridge/types/addresses.go

```diff
@@ -0,0 +1,7 @@
+package types
+
+import authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+
+var (
+	ModuleAddress = authtypes.NewModuleAddress(ModuleName)
```

Comment:
1. let's add a test to verify this value
2. nit: `BridgeModuleAddress` to be more explicit?

Reply:
I can't find the comment I made - maybe never got posted? But I prefer to keep this as ModuleAddress because 1. It follows the existing naming pattern mirrored by ModuleName, and 2. the implicit module is already specified by the package wherever it is imported. Ok to leave it?